dashboard/test/ui/runner.rb

```diff
@@ -25,6 +27,26 @@
 
 ENV['BUILD'] = `git rev-parse --short HEAD`
 
+S3_LOGS_BUCKET = 'cucumber-logs'
+S3_LOGS_PREFIX = GitUtils.current_branch
```

Comment:
Great improvement Brad! In the case where a developer runs a local `./runner.rb` against the test server, it looks like this will use the developer's local branch name as the bucket. Perhaps it would be better to use the remote server name as the prefix in the case where it's not a local run?